give.php

```diff
@@ -755,6 +758,25 @@ private function is_request( $type ) {
 			}
 		}
 
+		/**
+		 * Handle property get request
+		 *
+		 * @param string $propertyName
+		 *
+		 * @since 2.7.0
+		 * @return mixed
+		 */
+		function __get( $propertyName ) {
```

Comment:
One last thing here, so yourself (and everyone else) a favor and add this to the docblock for the class:
```php
@property-read Themes $themes
```
That will inform all IDE's that the read-only property exists and what it returns.

Reply:
https://github.com/impress-org/givewp/compare/40544c3ae37d...1d3d0f8d03b3